controller/linodemachine_controller.go

```diff
@@ -56,28 +66,33 @@ func (r *LinodeMachineReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	ctx, cancel := context.WithTimeout(ctx, reconciler.DefaultedLoopTimeout(r.ReconcileTimeout))
 	defer cancel()
 
-	log := ctrl.LoggerFrom(ctx)
+	log := ctrl.LoggerFrom(ctx).WithName("LinodeMachineReconciler").WithValues("name", req.NamespacedName.String())
 
-	// TODO(user): your logic here
 	linodeMachine := &infrav1.LinodeMachine{}
 	if err := r.Get(ctx, req.NamespacedName, linodeMachine); err != nil {
+		log.Info("Failed to fetch Linode machine", "error", err.Error())
+
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
 	machine, err := util.GetOwnerMachine(ctx, r.Client, linodeMachine.ObjectMeta)
 	if err != nil {
-		return ctrl.Result{}, err
-	}
-	if machine == nil {
+		log.Info("Failed to fetch owner machine", "error", err.Error())
+
+		return ctrl.Result{}, client.IgnoreNotFound(err)
```

Comment:
@luthermonson Does it make sense to requeu the object if owner machine is missing? Do we have a chance some other controller creates it in the background? Sorry I'm not sure.

Reply:
if something updates the owner refs later this should eventually get tripped so probably okay to ignore for now right?